lib/fastlane/plugin/sentry/actions/sentry_create_deploy.rb

```diff
@@ -0,0 +1,95 @@
+module Fastlane
+  module Actions
+    class SentryCreateDeployAction < Action
+      def self.run(params)
+        require 'shellwords'
+
+        Helper::SentryHelper.check_sentry_cli!
+        Helper::SentryConfig.parse_api_params(params)
+
+        version = params[:version]
+        version = "#{params[:app_identifier]}@#{params[:version]}" if params[:app_identifier]
```

Comment:
versions at sentry are usually `package@version+build`, eg `io.sentry.sample.iOS-Swift@7.1.3+1`
here I don't see a `build`, do we need to account for that?

Reply:
I'm not sure, this is the same code as in other actions. Maybe @philipphofmann or @HazAT can add more context?

Reply:
It's a question if they should match whats is sent by default, today the mobile SDKs send the SDK version by default as I've stated, so if they need to match, this would be required, maybe even adding to all the other actions too.

`package@version` without the `build` is also a valid release, but it's not what the SDKs send by default.

See https://docs.sentry.io/product/cli/releases/#creating-releases

Reply:
Good point @marandaneto, but I wouldn't worry about this right now. I created an issue for this https://github.com/getsentry/sentry-fastlane-plugin/issues/84